src/renderer/components/+workloads-deployments/deployment-scale-dialog.tsx

```diff
@@ -83,21 +83,45 @@ export class DeploymentScaleDialog extends Component<Props> {
     }
   }
 
+  desiredReplicasUp = () => {
+    this.desiredReplicas < this.scaleMax && this.desiredReplicas++
+  }
+  
+  desiredReplicasDown = () => {
+    this.desiredReplicas > 1 && this.desiredReplicas--
+  };
+
   renderContents() {
     const { currentReplicas, desiredReplicas, onChange, scaleMax } = this;
     const warning = currentReplicas < 10 && desiredReplicas > 90;
     return (
       <>
-        <div className="current-scale">
+        <div className="current-scale" data-testid="current-scale">
           <Trans>Current replica scale: {currentReplicas}</Trans>
         </div>
         <div className="flex gaps align-center">
-          <div className="desired-scale">
+          <div className="desired-scale" data-testid="desired-scale">
             <Trans>Desired number of replicas</Trans>: {desiredReplicas}
           </div>
           <div className="slider-container">
             <Slider value={desiredReplicas} max={scaleMax} onChange={onChange as any /** see: https://github.com/mui-org/material-ui/issues/20191 */}/>
           </div>
+          <div className="plus-minus-container">
+            <i
+              className="Icon material"
+              onClick={this.desiredReplicasUp}
+              data-testid="desired-replicas-up"
+            >
+              <span className="icon">add_circle_outline</span>
+            </i>
+            <i
+              className="Icon material"
+              onClick={this.desiredReplicasDown}
+              data-testid="desired-replicas-down"
+            >
+              <span className="icon">remove_circle_outline</span>
+            </i>
```

Comment:
For icons we're using custom `<Icon />` (`src/renderer/components/icon/icon.tsx`) component which contains some predefined behavior like built-in `tooltip` prop and icon's styling such as pointer cursor.

You can look for example of how to use it inside `src/renderer/components/+config-secrets/add-secret-dialog.tsx`.

So the first icon would look like
```
<Icon
  material="add_circle_outline"
  onClick={this.desiredReplicasUp}
  data-testid="desired-replicas-up"
/>
```

Reply:
done